### DISCUSSION TITLE
Self Learning

### DISCUSSION THREAD

**Discussion by dhruvateja1** [github/india#293] [Students]
I am unable to open any module in self learning. It's not opening.

**Comment by ArjunSrinivasanR** [comment]
Try switching to Incognito and refill in the details and all that. After making sure that the Orientation symbol is green, you can switch to the Self-learning tab. You can also try clearing cache and reloading the page or sign out and try again but this didnt work out for me. Make sue you apply for Github's Student Developer Pack.
